src/test/java/com/google/cloud/resourcemanager/ConstraintInfoTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.api.services.cloudresourcemanager.model.BooleanConstraint;
+import com.google.cloud.resourcemanager.ConstraintInfo.ListConstraint;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ConstraintInfoTest {
+  private static final String NAME = "constraints/serviceuser.services";
+  private static final String CONSTRAINT_DEFAULT = "ALLOW";
+  private static final String DISPLAY_NAME = "constraints-display-name";
+  private static final String DESCRIPTION =
+      "Detailed description of what this Constraint controls as well as how and where it is enforced";
+  private static final Integer VERSION = 1;
+
+  private static final ListConstraint LIST_CONSTRAINT = new ListConstraint("suggested-value", true);
+  private static ConstraintInfo CONSTRAINT_INFO;
+  private static BooleanConstraint BOOLEAN_CONSTRAINT;
+
+  @Before
+  public void setUp() {
+    BOOLEAN_CONSTRAINT = new BooleanConstraint();
```

Comment:
DANGER! Mutable static state in tests that can fail when tests are run in parallel. Non constants and mutable values should be instance fields.

Reply:
done